schema-bridge.graphql

```diff
@@ -277,9 +277,15 @@ type BridgeProtocol implements Protocol @entity {
 
   ##### Quantitative Data #####
 
-  " Current TVL (Total Value Locked) of the entire protocol on this chain. "
+  " Current TVL (Total Value Locked) of the entire protocol on this chain. This is the sum of totalValueExported + liquidity deposits "
   totalValueLockedUSD: BigDecimal!
 
+  " USD value of all assets bridged away from this chain. In other words, value of assets locked in contracts in this chain because they are liquid in some other chain. "
+  totalValueExportedUSD: BigDecimal!
```

Comment:
Although I agree that it would be more consistent, I think it would be more ambiguous. It could be mistaken with volume.

Whereas value imported/exported, while it can still be mistaken with volume, it is not related to value transferred, but to value living somewhere else, if that makes sense 😅 

Naming being naming

Reply:
makes sense. thanks!